lib/internal/webstreams/adapters.js

```diff
@@ -433,12 +434,21 @@ function newReadableStreamFromStreamReadable(streamReadable, options = kEmptyObj
       streamReadable);
   }
 
+  const streamType = options?.type;
+  if (streamType !== undefined) {
+    if (`${streamType}` !== 'bytes') {
+      throw new ERR_INVALID_ARG_VALUE('options.type', streamType);
+    }
+  }
```

Comment:
```suggestion
  validateObject(options, 'options');
  const { type: streamType } = options.type; 
  if (streamType !== undefined) {
    validateOneOf(options.type, 'options.type', ['bytes']);
  }
```

Better to use the validators from `lib/internal/validators.js` such as `validateObject` and `validateOneOf`